inst/include/interface/rcpp/rcpp_interface.hpp

```diff
@@ -669,12 +674,17 @@ RCPP_MODULE(fims) {
       .field("log_Fmort", &FleetInterface::log_Fmort)
       .field("nages", &FleetInterface::nages)
       .field("nyears", &FleetInterface::nyears)
+      .field("nlengths", &FleetInterface::nlengths)
```

Comment:
I assume it doesn't. All I did was add the "nlengths" row so it was already working with years and ages in the wrong order. I'm changing it though because I also find that annoying :) even if it doesn't make a difference.